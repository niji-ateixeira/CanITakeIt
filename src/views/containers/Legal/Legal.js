import React from 'react';
import { Text, View, Button } from 'react-native';
import { Actions } from 'react-native-router-flux';

import styles from './LegalStyles';

const Legal = () => (
  <View style={styles.container}>
    <Text>
        Legal!
    </Text>
    <Text>
        Legal.
    </Text>
    <Text>
        Legal.
    </Text>
    <Button
      onPress={Actions.pop}
      title="Learn More"
    />
  </View>
);

export default Legal;
