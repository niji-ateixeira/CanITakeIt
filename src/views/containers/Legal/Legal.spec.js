import React from 'react';
import Legal from './Legal';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Legal />).toJSON();
  expect(tree).toMatchSnapshot();
});
