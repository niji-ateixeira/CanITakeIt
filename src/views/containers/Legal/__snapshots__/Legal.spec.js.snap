// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Legal!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Legal.
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Legal.
  </Text>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={undefined}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Learn More
      </Text>
    </View>
  </View>
</View>
`;
